Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-alpine
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /web
+COPY . /web
+
+RUN apk update
+RUN apk add --no-cache --virtual .build-deps \
+    gcc \
```

Comment:
Can we split core and app dependencies into separate steps?

Reply:
I don't understand what you mean here. Could you explain what core and app dependencies are?

Reply:
Sorry, I should have been clearer.

By app deps, I meant python3-dev, postgresql-dev, ruby etc. (i.e. we need them to run python.org inside the container)

By core deps, I meant gcc, linux-headers, musl-dev etc. (i.e. we need them install our app dependencies)

Reply:
Will fix.